docs/source/tagset/elemento-day.rst

```diff
@@ -3,19 +3,19 @@
 <day>
 =====
 
-+----------------------------------+----------------------------------------------------------+
-| Aparece em                       | Ocorre                                                   |
-+==================================+==========================================================+
-| :ref:`elemento-pub-date`         | Zero ou uma vez nos elementos de :ref:`elemento-front`   |
-+----------------------------------+----------------------------------------------------------+
-| :ref:`elemento-product`          | Zero ou uma vez nos elementos de :ref:`elemento-back`    |
-+----------------------------------+----------------------------------------------------------+
-| :ref:`elemento-element-citation` | Zero ou uma vez nos elementos de :ref:`elemento-front`   |
-+----------------------------------+----------------------------------------------------------+
-| :ref:`elemento-date-in-citation` | Zero ou uma vez nos elementos de :ref:`elemento-front`   |
-+----------------------------------+----------------------------------------------------------+
-| :ref:`elemento-date`             | Zero ou uma vez nos elementos de :ref:`elemento-front`   |
-+----------------------------------+----------------------------------------------------------+
++----------------------------------+--------------------------------------------------------+
+| Aparece em                       | Ocorre                                                 |
++==================================+========================================================+
+| :ref:`elemento-pub-date`         | Zero ou uma vez nos elementos de :ref:`elemento-front` |
++----------------------------------+--------------------------------------------------------+
+| :ref:`elemento-product`          | Zero ou uma vez nos elementos de :ref:`elemento-back`  |
++----------------------------------+--------------------------------------------------------+
+| :ref:`elemento-element-citation` | Zero ou uma vez nos elementos de :ref:`elemento-front` |
++----------------------------------+--------------------------------------------------------+
+| :ref:`elemento-date-in-citation` | Zero ou uma vez nos elementos de :ref:`elemento-front` |
++----------------------------------+--------------------------------------------------------+
+| :ref:`elemento-date`             | Zero ou uma vez nos elementos de :ref:`elemento-front` |
++----------------------------------+--------------------------------------------------------+
```

Comment:
@francine-curivil perdeu o sentido escrever "em elementos de front ou back" já que em todos os casos o elemento aparece zero ou uma vez. Por favor manter a tabelinha assim:

Aparece em		Ocorre
<pub-date>		Zero ou uma vez 
<product>		Zero ou uma vez
<element-citation>Zero ou uma vez
<date-in-citation>	Zero ou uma vez
<date>			Zero ou uma vez